README.md

```diff
@@ -1 +1,6 @@
-tried: \>>>> [this](https://twitter.com/spacerezt/status/903047646549561344) <<<<
\ No newline at end of file
+tried: \>>>> [this](https://twitter.com/spacerezt/status/903047646549561344) <<<<
+
+todo: 
+- where to get images? 
+- improve spacing / centering of divs and imgs
```

Comment:
Were you thinking to have everything centered and the edges going into all directions all around?

I kinda like the flippy paper-ish edge on one side, but everthing everywhere would probably look good, too.

Reply:
Yeah I was originally thinking about having it all centered, but I agree the flippy paper-ish edge is good too